src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -311,11 +319,49 @@ public static void writeFileLines(Path file, List<String> lines) throws IOExcept
     public static void replaceInFile(Path file, String original, String replacement) throws IOException {
         InputStream inputStream = Files.newInputStream(file);
         List<String> lines = readFileLines(inputStream);
-        for (int i=0; i<lines.size(); i++)
+        for (int i = 0; i < lines.size(); i++) {
             lines.set(i, lines.get(i).replaceAll(original, replacement));
+        }
         writeFileLines(file, lines);
     }
 
+    /**
+     * Parses an xml file, looking for a given item within a given tag, and retrieves
+     * its value, if found
+     *
+     * @param fileName the full path of an xml file
+     * @param tag the tag name
+     * @param itemName the item name
```

Comment:
`itemName` refers to an XML attribute of the XML element tag? Perhaps rename it to `attributeName`?

Reply:
done